docs/intro.md

```diff
@@ -0,0 +1,122 @@
+# Introduction to Scipian
+
+## What is Scipian?
+
+Scipian is an open source platform to give teams a single, self-service place to
+access all the tools and services needed to develop and deploy applications. To
+put it more simply, it's a continuous delivery platform that combines disparate
+tools and utilities to create affinity around developing and deploying
+applications. It is not tools/platforms like GitHub Enterprise, Artifactory,
+Kubernetes, or OpenStack.
+
+The value stream behind Scipian is to focus on speed and agility - bringing new
+innovation and technology to support business strategy. Scipian is propelling
+the engineers to build applications by enabling engineers to focus on what
```

Comment:
nit: change the second `engineers` to `them`. `Scipian is propelling the engineers to build applications by enabling them to focus...`

Reply:
Feedback addressed in latest rebase.